airflow/providers/databricks/operators/databricks.py

```diff
@@ -206,6 +206,37 @@ class DatabricksSubmitRunOperator(BaseOperator):
         For more information on how to use this operator, take a look at the guide:
         :ref:`howto/operator:DatabricksSubmitRunOperator`
 
+    The last way is to use the param tasks to pass array of objects to instantiate this operator
+        tasks =[
+            {
+              "new_cluster": {
+                  "spark_version": "2.1.0-db3-scala2.11",
+                  "num_workers": 2
+            },
+              "notebook_task": {
+                  "notebook_path": "/Users/airflow@example.com/PrepareData"}}]
+
+        notebook_run = DatabricksSubmitRunOperator(
+            task_id='notebook_run',
+            tasks = tasks)
+
+    :param tasks: Array of Objects(RunSubmitTaskSettings) <= 100 items.
+        The supported params in the array are
+        - ``task_key``
+        - ``depends_on``
+        - ``existing_cluster_id``
+        - ``new_cluster``
+        - ``notebook_task``
+        - ``spark_jar_task``
+        - ``spark_python_task``
+        - ``spark_submit_task``
+        - ``pipeline_task``
+        - ``python_wheel_task``
+        - ``libraries``
+        - ``timeout_seconds``
```

Comment:
No need to list the options the link to the API is enough.
We don't want to maintain this list everytime there is a new option in the API.